src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -27,7 +27,7 @@
        To deal with that, we write this target which finds that added package, removes it from
        the references list, then adds it back in again with the 'EmbedInteropTypes' flag properly
        set. -->
-  <Target Name="MarkNuGetVisualStudioPackageWithEmbedInteropType" AfterTargets="ResolveNuGetPackageAssets">
+  <Target Name="MarkNuGetVisualStudioPackageWithEmbedInteropType" AfterTargets="ResolvePackageDependenciesForBuild">
     <ItemGroup>
       <_NuGetVisualStudioPackageReference Include="@(Reference)" Condition="'%(Reference.NuGetPackageId)' == 'NuGet.VisualStudio'" />
```

Comment:
Yes - actually it can be just '%(Filename)' == 'Nuget.VisualStudio'. I have updated the PR - hopefully that should fix it.